### DISCUSSION TITLE
MOES TS0601 TRV shows OTA update available. Maybe wrong model?

### DISCUSSION THREAD

**Discussion by kingrun** [Koenkk/zigbee2mqtt#10849] [Devices]
I got a message that my valves have an available OTA firmware update in the zigbee2mqtt UI. 

The valves are MOES TRV TS0601
In the dev console the info provided is this:
 Read result of 'genBasic': {"appVersion":85,"stackVersion":0,"hwVersion":1,"manufacturerName":"_TZE200_cwnjrr72","modelId":"TS0601"}
 
 I don't know whether I should update. As they were initially set up with the provided zigbee hub and the Tuya app didn't show any updates for these devices but zigbee2mqtt shows an update. 
 
 can anyone clarify if someone has this kind of TRV? I'm worried if it might break after updating.

**Comment by nikolavp** [comment]
I pulled the trigger on mine and it got updated so I guess there was a newer firmware. But yes, tuya didn't report this - because mine was paired with it previously.
